utility/src/main/java/org/oppia/util/parser/SvgEncoder.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.util.parser
+
+import com.bumptech.glide.load.EncodeStrategy
+import com.bumptech.glide.load.Options
+import com.bumptech.glide.load.ResourceEncoder
+import com.bumptech.glide.load.engine.Resource
+import com.caverock.androidsvg.SVG
+import java.io.File
+import java.io.FileOutputStream
+
+/** Encodes an SVG internal representation to {@link FileOutputStream}. */
+class SvgEncoder : ResourceEncoder<SVG?> {
+
+  override fun getEncodeStrategy(options: Options): EncodeStrategy {
+    return EncodeStrategy.SOURCE
+  }
+
+  override fun encode(data: Resource<SVG?>, file: File, options: Options): Boolean {
+    return try {
+      val svg: SVG = data.get()
+      val picture = svg.renderToPicture()
```

Comment:
Shouldn't we encode the actual SVG on disk rather than a bitmap representation of it? We want to retain the benefits of the image being scalable when we render it natively.

Reply:
It's loading picture into the disk and not the bitmap representation.